### PR TITLE
Feat email verification

### PR DESCRIPTION
**Pull request by roperzh**
OK to merge.

Short review.

Asana task link: https://app.asana.com/0/480796620059175/521587463583421

Changes in this PR:

- [x] Show errors directly from inkstone instead of hardcoding 'Username or Password Incorrect'
- [x] Add endpoints to `sdk-instone` featuring email confirmation and resend email confirmation
- [x] Add a different color to the 'user not verified' error 

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [ ] Did the [E2E checklist](https://haiku.quip.com/dGqeAEVSWdmg)
- [x] Expanded the [QA checklist](https://haiku.quip.com/rqwsAPsCGxqT)
- [x] Ran the automated tests and linted the code
- [ ] Wrote an automated test covering new functionality

### CODE REVIEW

File: packages/haiku-plumbing/src/Plumbing.js [hunk 1]
```diff
@@ -683,9 +683,15 @@ export default class Plumbing extends StateObject {
 
   authenticateUser (username, password, cb) {
     this.set('organizationName', null) // Unset this cache to avoid writing others folders if somebody switches accounts in the middle of a session
+    inkstone.setConfig({ baseUrl: 'https://localhost:8080/' })
```
Comment (stristr) [inline packages/haiku-plumbing/src/Plumbing.js]: Gotta remove this before okay to merge.

`.env` is the new way :smile:!

**Review by stristr — CHANGES_REQUESTED** [review]

File: packages/haiku-plumbing/src/Plumbing.js (hunk 1 above)
Comment (roperzh) [reply packages/haiku-plumbing/src/Plumbing.js]: damn! sorry about that

**Review by stristr — APPROVED** [review]
LGTM.

**Comment by stristr** [comment]
(Note: to fix lint issue, you just need to `// eslint-disable-next-line standard/no-callback-literal`.)

**Pull request merged**